libkbfs/key_manager_test.go

```diff
@@ -1638,6 +1670,14 @@ func testKeyManagerRekeyBit(t *testing.T, ver MetadataVer) {
 		t.Fatalf("Device 2 couldn't read a: %+v", err)
 	}
 
+	// Cancel the original promptPaper timer for user2 now that the
+	// TLF is readable; otherwise the rekey later on will hang because
+	// it looks like prompt for paper was already set.  TODO: should
```

Comment:
@songgao: what do you think about this TODO?  Are we going to have problems where device1 requests a rekey, then device2 completes the rekey and goes offline, then later device1 tries to rekey again for a new device, and has to wait for the timer to expire before it happens?